### PR TITLE
change examples to use latest sdk

### PR DESCRIPTION
**Pull request by alexstolr**

### CODE REVIEW

File: examples/example-js.js [hunk 1]
```diff
@@ -2,10 +2,11 @@ const { ClusterScanner, NonceScanner } = require('ssv-scanner');
 
 async function main() {
   const params = {
+    network: '',
     nodeUrl: '',
-    contractAddress: '',
     ownerAddress: '',
-    operatorIds: [],
+    contractAddress: '',
```
Comment (meshin-dev) [inline examples/example-js.js:8]: It looks like --network param already knows what contract to be used

**Review by meshin-dev — CHANGES_REQUESTED** [review]

**Review by axelrod-blox — APPROVED** [review]

File: examples/example-js.js (hunk 1 above)
Comment (alexstolr) [reply examples/example-js.js:8]: The example doesn't work without it, possibly some kind of bug in the code. looking into it.

Comment (alexstolr) [reply examples/example-js.js]: @meshin-dev fixed. Let's discuss offline.

**Review by meshin-dev — CHANGES_REQUESTED** [review]
@alexstolr you changed some core files, so build everything to vanilla js and push it as part of dist folder. 
This is for projects on js to be able to use sdk without typescript building process

File: src/lib/BaseScanner.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import Web3 from 'web3';
 
+// import {ContractProvider} from "./contract.provider";
```
[the remark this answers is not in the document]
Comment (alexstolr) [reply src/lib/BaseScanner.ts:3]: V

File: src/lib/BaseScanner.ts [hunk 3]
```diff
@@ -33,7 +34,7 @@ export abstract class BaseScanner {
     }
     this.params = scannerParams;
     // convert to checksum addresses
-    this.params.contractAddress = new Web3().utils.toChecksumAddress(this.params.contractAddress);
+
```
[the remark this answers is not in the document]
Comment (alexstolr) [reply src/lib/BaseScanner.ts:37]: V

File: src/lib/contract.provider.ts [hunk 4]
```diff
@@ -25,19 +25,20 @@ export class ContractProvider {
   private contract: ContractData;
   public web3: Web3;
 
-  constructor(contractEnv: string, contractGroup: string, nodeUrl: string) {
+  constructor(networkInfo: string, nodeUrl: string) {
```
[the remark this answers is not in the document]
Comment (alexstolr) [reply src/lib/contract.provider.ts:28]: It's network and env.

[the remark this answers is not in the document]
Comment (alexstolr) [reply src/lib/contract.provider.ts:28]: Changed

**Review by meshin-dev — APPROVED** [review]

**Pull request merged**